README.md

```diff
@@ -22,47 +22,86 @@ Re-implementation of Hamiltonian Generative Networks [paper](https://arxiv.org/a
 
 ## How to train
 The [train.py](train.py) script takes care of performing the training.
-To start a training, run
-```cmd
-python train.py [--params param_file] [--name experimen_name] [--cpu]
+To start a training, run 
+```commandline
+python train.py --train-config path_to_train_config_file
 ```
-where the optional argument `--param` can be used to specify the parameter
-file to be used. `--name` can be used to overwrite the `experiment_id` of the
-yaml file and save the data under the new name. `--cpu` can be used to force
-training on the CPU, otherwise the training will be performed in the GPU (if available). 
 
-Training can be done in on-line or off-line mode.
+```cmd
+optional arguments:
+  -h, --help            show this help message and exit
+  --train-config TRAIN_CONFIG
+                        Path to the training configuration yaml file.
+  --dataset-config DATASET_CONFIG
```

Comment:
Assert that either `--dataset-config`or  `--dataset-path are provided`. Maybe raise a warning if using some default yaml so that the user notices.